### DISCUSSION TITLE
Deal with contrast switcher when JS is off

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1612] [Ideas]
When JS is off, contrast switcher button is just a black 'A' on white background and isn't clickable.

**To discuss: how can we deal with this? what are our options?**

Contrast switcher when JS is off:
![image](https://user-images.githubusercontent.com/43404358/168620393-4e3c3ec6-7420-441f-864e-4a67ccf31afd.png)

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner @otis-bath @brockfanning thoughts please!

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @phillipgardiner @otis-bath Tech note here: the contrast switcher cannot work without javascript. So my guess would be that we should be hiding it entirely when javascript is off.

**Reply by phillipgardiner** [reply]
@LucyGwilliamAdmin @brockfanning @otis-bath Again, I'll refer back to brocks technical knowledge on this one, but if contrast switching cannot be provided in anyway via the use of HTML and possibly CSS then I guess it would have to be hidden. Again, I think we would need to adjust the interface (particularly on mobile) so you are not just left with a white gap where the icon would have been.
